lib/sorbet-rails/active_record_rbi_formatter.rb

```diff
@@ -127,13 +127,16 @@ def create_elem_specific_query_methods(class_rbi, type:, class_method:)
       )
     end
 
-    build_methods = %w(create create! new)
+    build_methods = %w(create create! new build first_or_create first_or_create! first_or_initialize)
     build_methods.each do |build_method|
+      # `build` method doesn't exist on the model, only on the relations
+      next if build_method == 'build' && class_method
+
       # This needs to match the generated method signature in activerecord.rbi and
       # in Rails 5.0 and 5.1 the param is a splat.
       if Rails.version =~ /^5\.(0|1)/ && %w(new build create create!).include?(build_method)
         param = Parameter.new("*args", type: "T.untyped")
-      elsif
+      else
```

Comment:
Oh was this a bug that I didn't notice? I already merged the other PR so lets try to merge this one quickly

Reply:
It definitely wasn't intended but it would still work. I think ruby evaluated the next line as the `elsif` statement but only if we didn't enter the first branch so it actually behaved the same.